### DISCUSSION TITLE
Extension request: help organize your conversations with ChatGPT

### DISCUSSION THREAD

**Discussion by francoisgervais** [adamlui/ai-web-extensions#91] [Ideas]
Hello,

Big fan of your widescreen extension, thought I would suggest an idea for another product. I create a lot of individual conversations in ChatGPT, and it's starting to be tough to keep track of the ones I want to keep. 

I would like to be able to favorite certain conversations so I can retrieve them easily in the future, perhaps with a section for them in the left panel: https://i.imgur.com/NRmi3a8.png

In a perfect world, also having folders or other additional organization tools would be neat too.

Hope you consider this, looking forward to your future work!

**Comment by adamlui** [comment]
Hey @francoisgervais thansk for enjoying Widescreen! I'm currently busy in the process of porting all my apps to Firefox/Chrome stores, but once I get thru all that, I can visit this idea to figure out how to do it then keep you updated

**Comment by nkormanik** [comment]
Creating Projects??  Would this partly solve the problem of organization???

Of course, every question does not fit nicely into a project.  Many are simply ad hoc questions.

However, creating various Projects, and populating those, seems to really improve things.

**Reply by francoisgervais** [reply]
This has in fact been solved by projects
